### PR TITLE
GDPR compliance

### PR DESCRIPTION
**Pull request by ecamellini**
See PR preview.

* Clicking on the `check` makes the banner disappear and saves the info in the cookie
* Privacy policy is linked in the banner and in the footer
* Ganalytics does not track IP's anymore
* Cookie expires in 12 months:
image

### CODE REVIEW

File: src/CookieBanner/CookieBanner.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  render() {
+    return (
+      <FlexView
+        className={cx('cookie-banner', { hidden: this.state.accepted })}
+        width='100%'
+        height={50}
+        hAlignContent='center'
+        vAlignContent='center'
+      >
+        By continuing your visit to this site, you accept the use of cookies by Google Analytics to make visits statistics.&nbsp;
```
Comment (danielegallingani) [inline src/CookieBanner/CookieBanner.js]: I'd be more vague here:
`By continuing your visit to this site you accept our Cookie Policy.`

Comment (ecamellini) [reply src/CookieBanner/CookieBanner.js]: `cookie and privacy policies`?

**Comment by gabro** [comment]
Shouldn't we also link the cookie policy in the footer?

**Comment by ecamellini** [comment]
@gabro true, I'll do it

**Comment by danielegallingani** [comment]
nope, it's linked in the privacy policy, there's no need for an additional link

File: src/CookieBanner/CookieBanner.js (hunk 1 above)
Comment (ecamellini) [reply src/CookieBanner/CookieBanner.js]: Done

**Pull request merged**